airflow/ui/src/components/Banner/BackfillBanner.tsx

```diff
@@ -0,0 +1,123 @@
+/*!
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import { Box, HStack, Spacer, Text } from "@chakra-ui/react";
+import { useState } from "react";
+import { MdClose, MdPause, MdPlayArrow, MdStop } from "react-icons/md";
+
+import {
+  useBackfillServiceCancelBackfill,
+  useBackfillServiceListBackfills,
+  useBackfillServicePauseBackfill,
+  useBackfillServiceUnpauseBackfill,
+} from "openapi/queries";
+
+import Time from "../Time";
+import { ProgressBar } from "../ui";
+import ActionButton from "../ui/ActionButton";
+
+type Props = {
+  readonly dagId: string;
+};
+
+const BackfillBanner = ({ dagId }: Props) => {
+  const { data, isLoading } = useBackfillServiceListBackfills({
```

Comment:
@dstandish Do you remember with this endpoint. Will cancelled backfills be returned?